### ISSUE TITLE
variables imported via `using` in previous notebooks are not visible

### ISSUE THREAD

**Issue by disberd** [disberd/PlutoDevMacros.jl#11]
Right now the package relies on `Base.names` to find out all the variables that are defined in the submodule containing the target of `@frompackage`.
This unfortunately does not expose the names of the variables that are imported in a module when simply `using` other packages.
See https://discourse.julialang.org/t/get-all-names-accessible-from-a-module/98492

This means that the behavior of @frompackage can not exactly mimic the chain of inclusions inside of the target Package source code.

Take as example the two notebooks [inner_notebook1.jl](https://github.com/disberd/PlutoDevMacros.jl/blob/71a2ba52f44ac4a5fe198f695ef946e2dc6c1447/test/TestPackage/src/inner_notebook1.jl) and [inner_notebook2.jl](https://github.com/disberd/PlutoDevMacros.jl/blob/71a2ba52f44ac4a5fe198f695ef946e2dc6c1447/test/TestPackage/src/inner_notebook2.jl) in the TestPackage

While inner_notebook2.jl should still se BenchmarkTools as it's imported within inner_notebook1.jl, this does not happen inside Pluto:
![image](https://user-images.githubusercontent.com/12846528/236875295-0bd807ea-5c9c-4e10-a891-e2832a2732d5.png)

The current workaround is to also explicitly use the package also inside `inner_notebook2.jl`. So translate the macrocall from
```julia
@fromparent import *
```
into 
```julia
@fromparent begin
    import *
    using >.BenchmarkTools
end
```

**Comment by disberd** [comment]
It seems this also applies to names explicitly imported with the synthax `using Package: name`. Doing `import Package: name` does instead make the imported name visible also when calling `using *` in `@fromparent`

**Comment by disberd** [comment]
Will be closed when #48 is merged. Check the PR for details.
